### DISCUSSION TITLE
Blueprints: WP CLI import flag to do a clean import

### DISCUSSION THREAD

**Discussion by blakewilson** [wpengine/atlas-content-modeler#483] [Ideas]
It could be helpful to provide a flag to clear WordPress of existing content before importing an ACM Blueprint. A few scenerios where this could be helpful:

* During development
* Building a "demo" site to showcase an ACM Blueprint

In both of these cases you are iteratively creating/modifying content for the ACM Blueprint export. In order to accurately see what content is including in the `acm-blueprint.zip`, you'll either have to reset the WP instance with a plugin, or wipe the database and start fresh.

Something like:

```
wp acm blueprint import <URL> --clean-import
```

Or the like would be really helpful in these scenerios.

**Comment by nickcernis** [comment]
@blakewilson Thanks for the suggestion! We were considering something like `wp acm blueprint reset` to restore ACM to its initial state.

Would you expect a flag like `--clean-import` or a command like `wp acm blueprint reset` to remove all content such as media, menus, and WP core posts and pages? Or just ACM content?
